src/main/java/dev/sigstore/rekor/client/RekorVerifier.java

```diff
@@ -90,6 +92,66 @@ public void verifyEntry(RekorEntry entry)
    * @throws RekorVerificationException if the entry cannot be verified
    */
   public void verifyInclusionProof(RekorEntry entry) throws RekorVerificationException {
-    throw new UnsupportedOperationException("Verifying inclusion proof is not yet supported");
+
+    var inclusionProof =
+        entry
+            .getVerification()
+            .getInclusionProof()
+            .orElseThrow(
+                () ->
+                    new RekorVerificationException(
+                        "No inclusion proof was found in the rekor entry"));
+
+    var leafHash =
+        Hashing.sha256()
+            .hashBytes(combineBytes(new byte[] {0x00}, Base64.getDecoder().decode(entry.getBody())))
+            .asBytes();
+
+    // see: https://datatracker.ietf.org/doc/rfc9162/ section 2.1.3.2
+    var fn = inclusionProof.getLogIndex();
```

Comment:
I find the spec algorithm variable names pretty unintuitive
were you tempted to rename them for the java implementation to something more java like?
you could add a comment or something to match them to the spec names

Reply:
sgtm, I'll update